docs/user/transitioning_from_defaults.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Transitionning from Anaconda's default packages"
+---
+
+Users wishing to transition from Anaconda's default channel should do so with
+care so as not to break their working environments. The first thing to realize
+is that conda-forge is incompatible with the packages provided in Anaconda's
+default channel. Often, the packages on conda-forge will require newer versions
+of a particular package, or, may simply be organized in a different fashion.
+The only workflow we can support as a community is one where the conda-forge
+channel takes priority over all other channels.
+
+If you were using `miniconda3` or `Anaconda3` and wish to move to use
+conda-forge, we outline two workflow. The first allows you to try packages from
+conda-forge without uninstalling your previous installation, the second
+involves uninstalling `miniconda3` or `Anaconda3`, and only using conda-forge
+channels.
+
+## Trying conda-forge in an isolated environment
+
+1. Create a conda-environment with very dependencies. The following command
+   will only use packages from conda-forge to install Python 3.11. In March
+   2024, Python 3.11, was chosen due to the fact that it has broad
+   compatibility with many packages in the conda-forge ecosystem.
+   You may adjust the version of python to the one of your liking.
+
+```bash
+conda create --name new_cf python=3.11 --channel conda-forge --override-channels
+```
+
+2. Activate your environment and add conda-forge with strict dependencies to
+   that particular environment.
+
+```bash
+    conda activate new_cf
+    conda config --env --add channels conda-forge
+    conda config --env --set channel_priority strict
+```
+
+3. Check the configuration of your conda environment.
+
+```bash
+conda config --show
+```
+
+   Should list out that conda-forge is the top most channel in your list.
+
+4. You are now in a position to use command likes
+
+```bash
+conda install my_desired_package_from_conda_forge
+```
+
+## Uninstalling Anaconda3 and installing miniforge3
+
+If you are ready to fully switch to conda-forge, we recommend uninstalling
+Anaconda3 and installing miniforge3. The following steps are guidelines on how
+to go about this, while backing up your work and your environments.
+
+1. Clean your existing conda installation. This should remove unused files from
+   your anaconda folder and make for smaller sized backup.
+
+```
+# The conda build purge command may fail if you don't have conda-build installed.
+# this is expected
+conda build purge
+conda clean --all
+```
+
+2. Create a zip archive of your conda installation. Use your favorite graphical tool
```

Comment:
They can also use `ls ~/.conda/environments.txt` (and/or `conda info --envs`) to show their environments, so they can `mv` them somewhere safe and then put them back in place. Or we could list how to generate platform-specific lockfiles with `conda list --explicit --md5` that they can then use to regenerate those envs.

Reply:
i'm mostly trying to keep it as simple as possible. 

Many of these users are true users of conda, and not reallly developers of it, so these "tricks" make it seem more scary than anything.

Trying to "keep an environment alive" like this, and changing the the default environment, is likely going to be a huge problem when they type `conda install ...` any other package.

Reply:
I wonder if it is worth mentioning `conda env export --from-history`. It is probably a simple way to get a close-ish env file for existing environments without much magic or advanced directory/file browsing/moving.

Reply:
Added a little paragraph for both things without sacrificing the simplicity of the zip archive. Let me know what you think.

Reply:
Perfect!